e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceMockingFlakyEMTest.java

```diff
@@ -48,6 +50,8 @@ public void externalServiceMockingTest() throws Throwable {
                     args.add("USER");
                     args.add("--externalServiceIP");
                     args.add("127.0.0.5");
+                    args.add("--minimize");
```

Comment:
if test disabled, shouldn't this be removed? u forgot to remove it, or was it added on purpose?

Reply:
fixed it